### PR TITLE
encoding for to assist GBIF SE

### PR DESCRIPTION
**Pull request by djtfmartin**
This PR ensures DBF files are correctly read and converted to UTF8

### CODE REVIEW

File: src/main/java/au/org/ala/layers/intersect/SimpleShapeFile.java [hunk 1]
```diff
@@ -1769,10 +1775,10 @@ public DBFRecord(ByteBuffer buffer, ArrayList<DBFField> fields) {
             try {
                 switch (f.getType()) {
                     case 'C':            //string
-                        record[i] = (new String(data, SimpleShapeFile.getDBFEncoding())).trim();
+                        record[i] = new String(new String(data, "ISO-8859-1").trim().getBytes("ISO-8859-1"),  StandardCharsets.UTF_8);
```
Comment (adam-collins) [inline src/main/java/au/org/ala/layers/intersect/SimpleShapeFile.java]: Use DBFField.convertToUTF8

File: src/main/java/au/org/ala/layers/intersect/SimpleShapeFile.java [hunk 2]
```diff
@@ -1769,10 +1775,10 @@ public DBFRecord(ByteBuffer buffer, ArrayList<DBFField> fields) {
...
                         break;
                     case 'N':            //number as string
-                        record[i] = (new String(data, SimpleShapeFile.getDBFEncoding())).trim();
+                        record[i] = new String(new String(data, "ISO-8859-1").trim().getBytes("ISO-8859-1"),  StandardCharsets.UTF_8);
```
Comment (adam-collins) [inline src/main/java/au/org/ala/layers/intersect/SimpleShapeFile.java]: Use DBFField.convertToUTF8

File: src/main/java/au/org/ala/layers/intersect/SimpleShapeFile.java (hunk 1 above)
Comment (djtfmartin) [reply src/main/java/au/org/ala/layers/intersect/SimpleShapeFile.java]: done

File: src/main/java/au/org/ala/layers/intersect/SimpleShapeFile.java (hunk 2 above)
Comment (djtfmartin) [reply src/main/java/au/org/ala/layers/intersect/SimpleShapeFile.java]: done

**Comment by djtfmartin** [comment]
thanks for review @adam-collins

**Pull request merged**